### ISSUE TITLE
Changed Judging groups and Award Groups but reports don't show on Judging Group

### ISSUE THREAD

**Issue by jeanniebadger** [jpschewe/fll-sw#1149]
I changed the Judging groups for several teams (DON"T ASK!!)

I changed both judging group and Award group to match

These changes seem to be correct in the software - Teams show up in corrected judging group for Subjective App and in Ref scoring.

The Report for Subjective by Category shows the change to Award group but not for Judging Group
NOTE:  Teams 3916, 50145, 3982, 3919, 4660, 3910 - these were changed in Pairs
[JudgingGroup and Award group don't agree subjectiveByCategoryAndTime.pdf](https://github.com/jpschewe/fll-sw/files/14045916/JudgingGroup.and.Award.group.don.t.agree.subjectiveByCategoryAndTime.pdf)
I will email the datafile

**Comment by jpschewe** [comment]
Looking at the admin page for judging groups and award groups:

- 3916 ag: Prairie jg: Prairie
- 50145 ag: Woods jg: Woods
- 3982 ag: Prairie jg: Prairie
- 3919 ag: Lakes jg: Lakes
- 4660 ag: Lakes jg: Lakes
- 3910 ag: Prairie jg: Prairie

SubjectiveByJudge agrees with the assignments in the admin page.

ScaledSubjectiveByAwardGroup is empty
I noticed that compute summarized scores didn't show anything either.

Looking in the database for judges I only see Vergene judging Core Values for Lakes and Marsh.

@jeanniebadger  there are no subjective scores in this database for the SPPS tournament. There's nothing to display on the report by award group. Everything looks fine otherwise.

**Comment by jeanniebadger** [comment]
If you look at the report I attached, you will see that the Judging Group and Award groups don't agree.

This is the report we use for check-in so they need to agree or it is confusing for the Check-in person and they may send the team to the wrong room.  I have attached the report I am talking about and highlighted the teams with issues

It seems like the report does not regenerate after the changes are made?

Vergene is a test judge.  I needed to verify that SPPS would let us login to server.  She is a teacher at Capitol Hill and tested for me.  That is why she is in datafile with no scores.
[JudgingGroup.and.Award.group.don.t.agree.subjectiveByCategoryAndTime.pdf](https://github.com/jpschewe/fll-sw/files/14058052/JudgingGroup.and.Award.group.don.t.agree.subjectiveByCategoryAndTime.pdf)
